en/complete_vehicles/README.md

```diff
@@ -8,7 +8,7 @@ This section contains information about complete vehicles that run PX4.
 - Consumer drones run PX4 natively/"out of the box":
   * Multicopter
     * [Teal One](https://tealdrones.com/teal-one/)
-    * [NXP HoverGames KIT-HGDRONEK66](https://www.nxp.com/KIT-HGDRONEK66) 
+    * [NXP HoverGames.com KIT-HGDRONEK66](https://www.nxp.com/KIT-HGDRONEK66) 
```

Comment:
Why ".com"? That looks really odd.

Reply:
@igalloway ^^^^

Reply:
I was thinking that this would make it clear to users they can actually go to HoverGames.com to find out more. But yes it could be removed since it looks odd.